### ISSUE TITLE
Contrast - podcast

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#105]
◦	All regular size blue links on the gray background have insufficient contrast (links in special episodes, “A11y Rules Podcast”, “*”).

**Comment by digilou** [comment]
Fixed in [e545846](https://github.com/nicolas-steenhout/main/commit/e5458464092429ecbb86b68ce583fadbc1e49380)
